frontend/catalyst/third_party/cuda/cuda_quantum.toml

```diff
@@ -48,3 +48,8 @@ mid_circuit_measurement = true
 # This field is currently unchecked but it is reserved for the purpose of
 # determining if the device supports dynamic qubit allocation/deallocation.
 dynamic_qubit_management = false
+
+# whether the device can support non-commuting measurements together 
+# in a single execution
+non_commuting_observables = true
```

Comment:
Have you added a test to cuda_quantum that verifies this transformation succeeds?

Reply:
Discussed offline: this change is currently unrelated to the cuda_quantum device, because it only applies to the new device API, and cuda_quantum uses the old API. 

It currently fails for circuits with non-commuting observables, because it attempts to diagonalize and expand assuming only commuting observables. I'll remove the flag for now, since it's not being used, and from the current behaviour of the device I'm not confident about whether or not the backend can natively support non-commuting observables or not. It will need to be updated when the CUDA device moves to the new device API.